lib/FieldType/Mapper/CheckboxFormMapper.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
+ */
+namespace EzSystems\RepositoryForms\FieldType\Mapper;
+
+use eZ\Publish\API\Repository\FieldTypeService;
+use EzSystems\RepositoryForms\Data\Content\FieldData;
+use EzSystems\RepositoryForms\FieldType\DataTransformer\FieldValueTransformer;
+use EzSystems\RepositoryForms\FieldType\FieldValueFormMapperInterface;
+use Symfony\Component\Form\FormInterface;
+
+class CheckboxFormMapper implements FieldValueFormMapperInterface
+{
+    /**
+     * @var \eZ\Publish\API\Repository\FieldTypeService
+     */
+    private $fieldTypeService;
+
+    public function __construct(FieldTypeService $fieldTypeService)
+    {
+        $this->fieldTypeService = $fieldTypeService;
+    }
+
+    public function mapFieldValueForm(FormInterface $fieldForm, FieldData $data)
```

Comment:
The signature of mapFieldValueForm method does not match the used signature in other mappers (for instance: [IntegerFormMapper](https://github.com/ezsystems/repository-forms/blob/master/lib/FieldType/Mapper/IntegerFormMapper.php#L19), [TextLineFormMapper](https://github.com/ezsystems/repository-forms/blob/master/lib/FieldType/Mapper/TextLineFormMapper.php#L34))
Would you agree on changing the variable name from **$fieldForm** to **$fieldDefinitionForm**?